layers/+tools/lsp/funcs.el

```diff
@@ -93,6 +93,7 @@
     "gk" #'spacemacs/lsp-avy-goto-word
     "gK" #'spacemacs/lsp-avy-goto-symbol
     "gM" #'lsp-ui-imenu
+    "gh" #'lsp-treemacs-call-hierarchy
```

Comment:
The layer already depends on `lsp-treemacs`: https://github.com/syl20bnr/spacemacs/blob/develop/layers/+tools/lsp/packages.el#L30

Reply:
@michaelpj 
but they can be disabled by user

Reply:
There are other uses of `lsp-treemacs` functions in this layer that are not guarded. I agree it would be an improvement to change that, but I'm not going to do that in this PR.

Reply:
you can leave others unchanged, but you need to guard this occurrence.

Reply:
done